public/controlpanel.php

```diff
@@ -287,8 +280,6 @@ function validateEmail() {
                 echo "<td>";
                 echo "<form method='POST' action='/request/user/update-motto.php'>";
                 echo "<input class='fullwidth' name='m' value=\"$userMottoString\" maxlength='49' id='usermottoinput' placeholder='Add your motto here! (No profanity please!)' />";
-                echo "<input type='hidden' name='u' VALUE='$user'>";
-                echo "<input type='hidden' name='c' VALUE='$cookie'>";
```

Comment:
This is manually passing the cookie information to the API page. It doesn't seem to be necessary as the real cookies are passed too. Many API pages rely only on the real cookies, so I think this is a safe change to make. It definitely works on Windows/Chrome.